src/vtquery.cpp

```diff
@@ -163,12 +190,97 @@ struct CompareDistance {
     }
 };
 
+std::vector<vtzero::index_value_pair> get_comparison_tags(vtzero::feature f) {
+    std::vector<vtzero::index_value_pair> v;
+    while (auto ii = f.next_property_indexes()) {
+        v.push_back(ii);
```

Comment:
The compiler might already be doing this, but to be sure a copy is not made I would recommend being explicit: `v.push_back(std::move(ii))`;